src/content/blog/astro-4150.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: "Astro 4.15"
+description: "Astro 4.15 is now available! This release stabilize Astro Actions - our solution for fully type-safe backend functions. Also included: support for LibSQL remotes in Astro DB, a new timeout option for `client:idle`, and more."
```

Comment:
wording nit + em dash instead of hyphen
```suggestion
description: "Astro 4.15 is now available! This release stabilizes Astro Actions — our solution for fully type-safe backend functions. Also included: support for LibSQL remotes in Astro DB, a new timeout option for `client:idle`, and more."
```